class/defaults.yml

```diff
@@ -48,6 +52,20 @@ parameters:
           cpu: 200m
           memory: 200Mi
 
+    slos:
```

Comment:
This is an acronym for? sli exporter?

Reply:
Service Level Objectives? The intention would be that we'll add other SLO relevant parameters later on